.github/workflows/cd.yml

```diff
@@ -0,0 +1,29 @@
+name: CD
```

Comment:
✅ This is great . Should we though migrate it to our internal repo ?

Reply:
@toninis  Sure . I'l raise a seperate ticket to tackle this.